actionview/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+*   Add `include_seconds` option for `time_field`
+
+        <%= form.time_field :foo, include_seconds: false %>
+        <%# => <input value="16:22" type="time" />
```

Comment:
```suggestion
        <%# => <input value="16:22" type="time" /> %>
```

Reply:
I think `# => <input value="16:22" type="time" />` might actually be correct. Will fix this + the entry I copied the bad syntax from.

Reply:
Yeah, that would read even better, I think 👍